Muelu: add region MG examples

mayrmt opened the pull request:
@trilinos/muelu 
@lucbv @pohm01 @rstumin 

## Motivation

Reproduce the examples, that we will include in the paper on region MG.

## Related Issues

* Follows the source code restructuring done in PR #7830 


## Stakeholder Feedback

This work is part of an ASCR project with @rstumin and @lucbv.

## Testing

All tests pass locally.

mayrmt commented:
@pohm01 Can you please add a prototype of the structured/unstructured example to this branch? Thanks!

pohm01 commented on packages/muelu/research/regionMG/example/CMakeLists.txt:
Do we want to specify `--nz` as well?

mayrmt commented on packages/muelu/research/regionMG/example/CMakeLists.txt:
@pohm01 Good point! Since this is an example, we should definitely specify that to be more clear about the setup. I already pushed the necessary change.

mayrmt commented:
@pohm01 @lucbv I added one more example (Poisson with AMG as coarse solver) and cleaned up the xml-files. From my point of view, this is ready to be merged. Let me know, what you think!

trilinos-autotester commented:
Status Flag 'Pre-Test Inspection' - Auto Inspected - Inspection Is Not Necessary for this Pull Request.

trilinos-autotester commented:
Status Flag 'Pull Request AutoTester' - Testing Jenkins Projects:

<details>
<summary>
  <b>Pull Request Auto Testing STARTING</b> (click to expand)
</summary>

<h2>Build Information</h2>
<h3>Test Name: Trilinos_pullrequest_gcc_4.8.4</h3>
<ul>
  <li>Build Num: 7684</li>
  <li>Status: STARTED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>COMPILER_MODULE</td>
    <td>sems-gcc/4.8.4</td>
  </tr>

  <tr>
    <td>JENKINS_BUILD_TYPE</td>
    <td>Release</td>
  </tr>

  <tr>
    <td>JENKINS_COMM_TYPE</td>
    <td>MPI</td>
  </tr>

  <tr>
    <td>JENKINS_DO_COMPLEX</td>
    <td>OFF</td>
  </tr>

  <tr>
    <td>JENKINS_JOB_TYPE</td>
    <td>Experimental</td>
  </tr>

  <tr>
    <td>MPI_MODULE</td>
    <td>sems-openmpi/1.8.7</td>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td>pkg: MueLu</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>7908</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>TRILINOS</td>
  </tr>

  <tr>
    <td>TRILINOS_SOURCE_BRANCH</td>
    <td>muelu-add-regionMG-examples</td>
  </tr>

  <tr>
    <td>TRILINOS_SOURCE_REPO</td>
    <td>https://github.com/mayrmt/Trilinos</td>
  </tr>

  <tr>
    <td>TRILINOS_SOURCE_SHA</td>
    <td>99bbf575b65d7d2d0388964bf4325650e6264977</td>
  </tr>

  <tr>
    <td>TRILINOS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>TRILINOS_TARGET_REPO</td>
    <td>https://github.com/trilinos/Trilinos</td>
  </tr>

  <tr>
    <td>TRILINOS_TARGET_SHA</td>
    <td>4c049b498673f206ec75cfe5b6199f3758f52ab5</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: Trilinos_pullrequest_intel_17.0.1</h3>
<ul>
  <li>Build Num: 7494</li>
  <li>Status: STARTED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td>pkg: MueLu</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>7908</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>TRILINOS</td>
  </tr>

  <tr>
    <td>TRILINOS_SOURCE_BRANCH</td>
    <td>muelu-add-regionMG-examples</td>
  </tr>

  <tr>
    <td>TRILINOS_SOURCE_REPO</td>
    <td>https://github.com/mayrmt/Trilinos</td>
  </tr>

  <tr>
    <td>TRILINOS_SOURCE_SHA</td>
    <td>99bbf575b65d7d2d0388964bf4325650e6264977</td>
  </tr>

  <tr>
    <td>TRILINOS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>TRILINOS_TARGET_REPO</td>
    <td>https://github.com/trilinos/Trilinos</td>
  </tr>

  <tr>
    <td>TRILINOS_TARGET_SHA</td>
    <td>4c049b498673f206ec75cfe5b6199f3758f52ab5</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: Trilinos_pullrequest_gcc_4.9.3_SERIAL</h3>
<ul>
  <li>Build Num: 5919</li>
  <li>Status: STARTED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td>pkg: MueLu</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>7908</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>TRILINOS</td>
  </tr>

  <tr>
    <td>TRILINOS_SOURCE_BRANCH</td>
    <td>muelu-add-regionMG-examples</td>
  </tr>

  <tr>
    <td>TRILINOS_SOURCE_REPO</td>
    <td>https://github.com/mayrmt/Trilinos</td>
  </tr>

  <tr>
    <td>TRILINOS_SOURCE_SHA</td>
    <td>99bbf575b65d7d2d0388964bf4325650e6264977</td>
  </tr>

  <tr>
    <td>TRILINOS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>TRILINOS_TARGET_REPO</td>
    <td>https://github.com/trilinos/Trilinos</td>
  </tr>

  <tr>
    <td>TRILINOS_TARGET_SHA</td>
    <td>4c049b498673f206ec75cfe5b6199f3758f52ab5</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: Trilinos_pullrequest_gcc_7.2.0_debug</h3>
<ul>
  <li>Build Num: 139</li>
  <li>Status: STARTED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td>pkg: MueLu</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>7908</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>TRILINOS</td>
  </tr>

  <tr>
    <td>TRILINOS_SOURCE_BRANCH</td>
    <td>muelu-add-regionMG-examples</td>
  </tr>

  <tr>
    <td>TRILINOS_SOURCE_REPO</td>
    <td>https://github.com/mayrmt/Trilinos</td>
  </tr>

  <tr>
    <td>TRILINOS_SOURCE_SHA</td>
    <td>99bbf575b65d7d2d0388964bf4325650e6264977</td>
  </tr>

  <tr>
    <td>TRILINOS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>TRILINOS_TARGET_REPO</td>
    <td>https://github.com/trilinos/Trilinos</td>
  </tr>

  <tr>
    <td>TRILINOS_TARGET_SHA</td>
    <td>4c049b498673f206ec75cfe5b6199f3758f52ab5</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: Trilinos_pullrequest_gcc_8.3.0</h3>
<ul>
  <li>Build Num: 1955</li>
  <li>Status: STARTED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td>pkg: MueLu</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>7908</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>TRILINOS</td>
  </tr>

  <tr>
    <td>TRILINOS_SOURCE_BRANCH</td>
    <td>muelu-add-regionMG-examples</td>
  </tr>

  <tr>
    <td>TRILINOS_SOURCE_REPO</td>
    <td>https://github.com/mayrmt/Trilinos</td>
  </tr>

  <tr>
    <td>TRILINOS_SOURCE_SHA</td>
    <td>99bbf575b65d7d2d0388964bf4325650e6264977</td>
  </tr>

  <tr>
    <td>TRILINOS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>TRILINOS_TARGET_REPO</td>
    <td>https://github.com/trilinos/Trilinos</td>
  </tr>

  <tr>
    <td>TRILINOS_TARGET_SHA</td>
    <td>4c049b498673f206ec75cfe5b6199f3758f52ab5</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: Trilinos_pullrequest_cuda_9.2</h3>
<ul>
  <li>Build Num: 5251</li>
  <li>Status: STARTED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td>pkg: MueLu</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>7908</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>TRILINOS</td>
  </tr>

  <tr>
    <td>TRILINOS_SOURCE_BRANCH</td>
    <td>muelu-add-regionMG-examples</td>
  </tr>

  <tr>
    <td>TRILINOS_SOURCE_REPO</td>
    <td>https://github.com/mayrmt/Trilinos</td>
  </tr>

  <tr>
    <td>TRILINOS_SOURCE_SHA</td>
    <td>99bbf575b65d7d2d0388964bf4325650e6264977</td>
  </tr>

  <tr>
    <td>TRILINOS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>TRILINOS_TARGET_REPO</td>
    <td>https://github.com/trilinos/Trilinos</td>
  </tr>

  <tr>
    <td>TRILINOS_TARGET_SHA</td>
    <td>4c049b498673f206ec75cfe5b6199f3758f52ab5</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: Trilinos_pullrequest_clang_10.0.0</h3>
<ul>
  <li>Build Num: 400</li>
  <li>Status: STARTED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td>pkg: MueLu</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>7908</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>TRILINOS</td>
  </tr>

  <tr>
    <td>TRILINOS_SOURCE_BRANCH</td>
    <td>muelu-add-regionMG-examples</td>
  </tr>

  <tr>
    <td>TRILINOS_SOURCE_REPO</td>
    <td>https://github.com/mayrmt/Trilinos</td>
  </tr>

  <tr>
    <td>TRILINOS_SOURCE_SHA</td>
    <td>99bbf575b65d7d2d0388964bf4325650e6264977</td>
  </tr>

  <tr>
    <td>TRILINOS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>TRILINOS_TARGET_REPO</td>
    <td>https://github.com/trilinos/Trilinos</td>
  </tr>

  <tr>
    <td>TRILINOS_TARGET_SHA</td>
    <td>4c049b498673f206ec75cfe5b6199f3758f52ab5</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: Trilinos_pullrequest_python_2</h3>
<ul>
  <li>Build Num: 3251</li>
  <li>Status: STARTED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td>pkg: MueLu</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>7908</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>TRILINOS</td>
  </tr>

  <tr>
    <td>TRILINOS_SOURCE_BRANCH</td>
    <td>muelu-add-regionMG-examples</td>
  </tr>

  <tr>
    <td>TRILINOS_SOURCE_REPO</td>
    <td>https://github.com/mayrmt/Trilinos</td>
  </tr>

  <tr>
    <td>TRILINOS_SOURCE_SHA</td>
    <td>99bbf575b65d7d2d0388964bf4325650e6264977</td>
  </tr>

  <tr>
    <td>TRILINOS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>TRILINOS_TARGET_REPO</td>
    <td>https://github.com/trilinos/Trilinos</td>
  </tr>

  <tr>
    <td>TRILINOS_TARGET_SHA</td>
    <td>4c049b498673f206ec75cfe5b6199f3758f52ab5</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: Trilinos_pullrequest_python_3</h3>
<ul>
  <li>Build Num: 3272</li>
  <li>Status: STARTED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td>pkg: MueLu</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>7908</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>TRILINOS</td>
  </tr>

  <tr>
    <td>TRILINOS_SOURCE_BRANCH</td>
    <td>muelu-add-regionMG-examples</td>
  </tr>

  <tr>
    <td>TRILINOS_SOURCE_REPO</td>
    <td>https://github.com/mayrmt/Trilinos</td>
  </tr>

  <tr>
    <td>TRILINOS_SOURCE_SHA</td>
    <td>99bbf575b65d7d2d0388964bf4325650e6264977</td>
  </tr>

  <tr>
    <td>TRILINOS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>TRILINOS_TARGET_REPO</td>
    <td>https://github.com/trilinos/Trilinos</td>
  </tr>

  <tr>
    <td>TRILINOS_TARGET_SHA</td>
    <td>4c049b498673f206ec75cfe5b6199f3758f52ab5</td>
  </tr>
</table>

<h2>Using Repos:</h2>

<dl>
  <dt>Repo: TRILINOS (mayrmt/Trilinos)</dt>
  <ul>
    <li>Branch: muelu-add-regionMG-examples</li>
    <li>SHA: 99bbf575b65d7d2d0388964bf4325650e6264977</li>
    <li>Mode: TEST_REPO</li>
  </ul>
</dl>

Pull Request Author: mayrmt
</details>

trilinos-autotester commented:
Status Flag 'Pull Request AutoTester' - Jenkins Testing: all Jobs PASSED

<details>
<summary>
  <b>Pull Request Auto Testing has PASSED</b> (click to expand)
</summary>

<h2>Build Information</h2>
<h3>Test Name: Trilinos_pullrequest_gcc_4.8.4</h3>
<ul>
  <li>Build Num: 7684</li>
  <li>Status: PASSED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>COMPILER_MODULE</td>
    <td>sems-gcc/4.8.4</td>
  </tr>

  <tr>
    <td>JENKINS_BUILD_TYPE</td>
    <td>Release</td>
  </tr>

  <tr>
    <td>JENKINS_COMM_TYPE</td>
    <td>MPI</td>
  </tr>

  <tr>
    <td>JENKINS_DO_COMPLEX</td>
    <td>OFF</td>
  </tr>

  <tr>
    <td>JENKINS_JOB_TYPE</td>
    <td>Experimental</td>
  </tr>

  <tr>
    <td>MPI_MODULE</td>
    <td>sems-openmpi/1.8.7</td>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td>pkg: MueLu</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>7908</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>TRILINOS</td>
  </tr>

  <tr>
    <td>TRILINOS_SOURCE_BRANCH</td>
    <td>muelu-add-regionMG-examples</td>
  </tr>

  <tr>
    <td>TRILINOS_SOURCE_REPO</td>
    <td>https://github.com/mayrmt/Trilinos</td>
  </tr>

  <tr>
    <td>TRILINOS_SOURCE_SHA</td>
    <td>99bbf575b65d7d2d0388964bf4325650e6264977</td>
  </tr>

  <tr>
    <td>TRILINOS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>TRILINOS_TARGET_REPO</td>
    <td>https://github.com/trilinos/Trilinos</td>
  </tr>

  <tr>
    <td>TRILINOS_TARGET_SHA</td>
    <td>4c049b498673f206ec75cfe5b6199f3758f52ab5</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: Trilinos_pullrequest_intel_17.0.1</h3>
<ul>
  <li>Build Num: 7494</li>
  <li>Status: PASSED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td>pkg: MueLu</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>7908</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>TRILINOS</td>
  </tr>

  <tr>
    <td>TRILINOS_SOURCE_BRANCH</td>
    <td>muelu-add-regionMG-examples</td>
  </tr>

  <tr>
    <td>TRILINOS_SOURCE_REPO</td>
    <td>https://github.com/mayrmt/Trilinos</td>
  </tr>

  <tr>
    <td>TRILINOS_SOURCE_SHA</td>
    <td>99bbf575b65d7d2d0388964bf4325650e6264977</td>
  </tr>

  <tr>
    <td>TRILINOS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>TRILINOS_TARGET_REPO</td>
    <td>https://github.com/trilinos/Trilinos</td>
  </tr>

  <tr>
    <td>TRILINOS_TARGET_SHA</td>
    <td>4c049b498673f206ec75cfe5b6199f3758f52ab5</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: Trilinos_pullrequest_gcc_4.9.3_SERIAL</h3>
<ul>
  <li>Build Num: 5919</li>
  <li>Status: PASSED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td>pkg: MueLu</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>7908</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>TRILINOS</td>
  </tr>

  <tr>
    <td>TRILINOS_SOURCE_BRANCH</td>
    <td>muelu-add-regionMG-examples</td>
  </tr>

  <tr>
    <td>TRILINOS_SOURCE_REPO</td>
    <td>https://github.com/mayrmt/Trilinos</td>
  </tr>

  <tr>
    <td>TRILINOS_SOURCE_SHA</td>
    <td>99bbf575b65d7d2d0388964bf4325650e6264977</td>
  </tr>

  <tr>
    <td>TRILINOS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>TRILINOS_TARGET_REPO</td>
    <td>https://github.com/trilinos/Trilinos</td>
  </tr>

  <tr>
    <td>TRILINOS_TARGET_SHA</td>
    <td>4c049b498673f206ec75cfe5b6199f3758f52ab5</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: Trilinos_pullrequest_gcc_7.2.0_debug</h3>
<ul>
  <li>Build Num: 139</li>
  <li>Status: PASSED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td>pkg: MueLu</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>7908</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>TRILINOS</td>
  </tr>

  <tr>
    <td>TRILINOS_SOURCE_BRANCH</td>
    <td>muelu-add-regionMG-examples</td>
  </tr>

  <tr>
    <td>TRILINOS_SOURCE_REPO</td>
    <td>https://github.com/mayrmt/Trilinos</td>
  </tr>

  <tr>
    <td>TRILINOS_SOURCE_SHA</td>
    <td>99bbf575b65d7d2d0388964bf4325650e6264977</td>
  </tr>

  <tr>
    <td>TRILINOS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>TRILINOS_TARGET_REPO</td>
    <td>https://github.com/trilinos/Trilinos</td>
  </tr>

  <tr>
    <td>TRILINOS_TARGET_SHA</td>
    <td>4c049b498673f206ec75cfe5b6199f3758f52ab5</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: Trilinos_pullrequest_gcc_8.3.0</h3>
<ul>
  <li>Build Num: 1955</li>
  <li>Status: PASSED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td>pkg: MueLu</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>7908</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>TRILINOS</td>
  </tr>

  <tr>
    <td>TRILINOS_SOURCE_BRANCH</td>
    <td>muelu-add-regionMG-examples</td>
  </tr>

  <tr>
    <td>TRILINOS_SOURCE_REPO</td>
    <td>https://github.com/mayrmt/Trilinos</td>
  </tr>

  <tr>
    <td>TRILINOS_SOURCE_SHA</td>
    <td>99bbf575b65d7d2d0388964bf4325650e6264977</td>
  </tr>

  <tr>
    <td>TRILINOS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>TRILINOS_TARGET_REPO</td>
    <td>https://github.com/trilinos/Trilinos</td>
  </tr>

  <tr>
    <td>TRILINOS_TARGET_SHA</td>
    <td>4c049b498673f206ec75cfe5b6199f3758f52ab5</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: Trilinos_pullrequest_cuda_9.2</h3>
<ul>
  <li>Build Num: 5251</li>
  <li>Status: PASSED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td>pkg: MueLu</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>7908</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>TRILINOS</td>
  </tr>

  <tr>
    <td>TRILINOS_SOURCE_BRANCH</td>
    <td>muelu-add-regionMG-examples</td>
  </tr>

  <tr>
    <td>TRILINOS_SOURCE_REPO</td>
    <td>https://github.com/mayrmt/Trilinos</td>
  </tr>

  <tr>
    <td>TRILINOS_SOURCE_SHA</td>
    <td>99bbf575b65d7d2d0388964bf4325650e6264977</td>
  </tr>

  <tr>
    <td>TRILINOS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>TRILINOS_TARGET_REPO</td>
    <td>https://github.com/trilinos/Trilinos</td>
  </tr>

  <tr>
    <td>TRILINOS_TARGET_SHA</td>
    <td>4c049b498673f206ec75cfe5b6199f3758f52ab5</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: Trilinos_pullrequest_clang_10.0.0</h3>
<ul>
  <li>Build Num: 400</li>
  <li>Status: PASSED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td>pkg: MueLu</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>7908</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>TRILINOS</td>
  </tr>

  <tr>
    <td>TRILINOS_SOURCE_BRANCH</td>
    <td>muelu-add-regionMG-examples</td>
  </tr>

  <tr>
    <td>TRILINOS_SOURCE_REPO</td>
    <td>https://github.com/mayrmt/Trilinos</td>
  </tr>

  <tr>
    <td>TRILINOS_SOURCE_SHA</td>
    <td>99bbf575b65d7d2d0388964bf4325650e6264977</td>
  </tr>

  <tr>
    <td>TRILINOS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>TRILINOS_TARGET_REPO</td>
    <td>https://github.com/trilinos/Trilinos</td>
  </tr>

  <tr>
    <td>TRILINOS_TARGET_SHA</td>
    <td>4c049b498673f206ec75cfe5b6199f3758f52ab5</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: Trilinos_pullrequest_python_2</h3>
<ul>
  <li>Build Num: 3251</li>
  <li>Status: PASSED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td>pkg: MueLu</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>7908</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>TRILINOS</td>
  </tr>

  <tr>
    <td>TRILINOS_SOURCE_BRANCH</td>
    <td>muelu-add-regionMG-examples</td>
  </tr>

  <tr>
    <td>TRILINOS_SOURCE_REPO</td>
    <td>https://github.com/mayrmt/Trilinos</td>
  </tr>

  <tr>
    <td>TRILINOS_SOURCE_SHA</td>
    <td>99bbf575b65d7d2d0388964bf4325650e6264977</td>
  </tr>

  <tr>
    <td>TRILINOS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>TRILINOS_TARGET_REPO</td>
    <td>https://github.com/trilinos/Trilinos</td>
  </tr>

  <tr>
    <td>TRILINOS_TARGET_SHA</td>
    <td>4c049b498673f206ec75cfe5b6199f3758f52ab5</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: Trilinos_pullrequest_python_3</h3>
<ul>
  <li>Build Num: 3272</li>
  <li>Status: PASSED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td>pkg: MueLu</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>7908</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>TRILINOS</td>
  </tr>

  <tr>
    <td>TRILINOS_SOURCE_BRANCH</td>
    <td>muelu-add-regionMG-examples</td>
  </tr>

  <tr>
    <td>TRILINOS_SOURCE_REPO</td>
    <td>https://github.com/mayrmt/Trilinos</td>
  </tr>

  <tr>
    <td>TRILINOS_SOURCE_SHA</td>
    <td>99bbf575b65d7d2d0388964bf4325650e6264977</td>
  </tr>

  <tr>
    <td>TRILINOS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>TRILINOS_TARGET_REPO</td>
    <td>https://github.com/trilinos/Trilinos</td>
  </tr>

  <tr>
    <td>TRILINOS_TARGET_SHA</td>
    <td>4c049b498673f206ec75cfe5b6199f3758f52ab5</td>
  </tr>
</table>
</details>


<br>[CDash Test Results for PR# 7908.](https://testing.sandia.gov/cdash/index.php?project=Trilinos&filtercombine=and&filtercombine=and&filtercombine=and&filtercombine=and&filtercombine=and&filtercombine=and&filtercount=2&showfilters=1&filtercombine=and&field1=buildname&compare1=63&value1=PR-7908-test&field2=buildstarttime&compare2=84&value2=NOW
)<br>

trilinos-autotester commented:
Status Flag 'Pre-Merge Inspection' - - This Pull Request Requires Inspection... The code must be inspected by a member of the Team before Testing/Merging
           THE LAST COMMIT TO THIS PULL REQUEST HAS BEEN REVIEWED, BUT NOT ACCEPTED OR REQUIRES CHANGES

trilinos-autotester commented:
All Jobs Finished; status = PASSED, However Inspection must be performed before merge can occur...

trilinos-autotester commented:
Status Flag 'Pre-Merge Inspection' - SUCCESS: The last commit to this Pull Request has been INSPECTED AND APPROVED by [ lucbv ]!

trilinos-autotester commented:
Status Flag 'Pull Request AutoTester' - AutoMerge IS ENABLED, but the Label AT: AUTOMERGE is not set.  Either set Label AT: AUTOMERGE or manually merge the PR...
